src/analyzer.py

```diff
@@ -138,3 +140,38 @@ def noun_phrase(input_text):
     for chunk in doc.noun_chunks:
         n_phrase_lst.append(str(chunk))
     return n_phrase_lst
+
+
+def senti(tokens_column, sign):
+    """Create columns for positive and negative words"""
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        words = create_word_list(token_element)
+        # Add the words at the end of the list to the display series since
+        # they have the most positive sentiment value
+        if sign == 1:
+            display_series.append(", ".join(words[::-1][0:3]))
+        else:
+            display_series.append(", ".join(words[0:3]))
+    # Return an entire series based on the display_series list
+    return pd.Series(display_series)
+
+
+def create_word_list(token_element):
+    """Creates and sorts a word list from a list of tokens"""
+    # Convert the token list into a set so that it only has the unique words
+    words = set(token_element)
+    # Convert back into list to iterate through
+    unique_words = list(words)
+    # Insertion sort algorithm using the polarity to compare the words
+    for i in range(len(unique_words)):
+        key = unique_words[i]
+        j = i - 1
+        while j >= 0 and TextBlob(unique_words[j]).sentiment.polarity \
+                > TextBlob(key).sentiment.polarity:
+            unique_words[j + 1] = unique_words[j]
+            j -= 1
+        unique_words[j + 1] = key
```

Comment:
I appreciate the effort putting into implementing the insertion sort. A quicker and cleaners solution might be just to use the `sort()` function, where the key is the polarity score.

See here: https://realpython.com/python-sort/#sorted-with-a-key-argument